.github/workflows/check.yml

```diff
@@ -17,14 +17,15 @@ jobs:
     runs-on: ${{ matrix.system }}
     container: ${{ matrix.container }}
     steps:
-      - uses: actions/checkout@v2
-      - name: install git
+      - name: install deps
         if: matrix.container == 'ubuntu:18.04'
         run: |
           apt update -y
-          apt install software-properties-common -y
+          apt install -y software-properties-common
           add-apt-repository ppa:git-core/ppa
-          apt -y update && apt install -y git
+          apt update -y
```

Comment:
![](https://orly-appstore.herokuapp.com/generate?title=Mastering%20tautological%20redundancy&top_text=Like%20if%20apt%20update%20--yes%20had%20super%20cow%20powers!&author=Blamen%20von%20Blinkenlichten&image_code=31&theme=6&guide_text=The%20self-evident%20guide&guide_text_placement=bottom_right)

Reply:
Do we really need the `-y` flag for the `apt update` commands? 🙊